fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -214,11 +216,28 @@ protected Response getContent(final String rangeValue,
         return ok(outputStream).build();
     }
 
+
+    protected URI getExternalResourceLocation(final MediaType mediaType) throws UnsupportedAccessTypeException {
```

Comment:
@awoods It would be, if it didn't mean adding the javax.ws.rs-api dependency to the fcrepo-kernel-api pom in order to make use of MediaType.   It seems to me that since the method is only used in the fcrepo-http-api layer.  maybe we should keep it as is.   I'm willing to change it if you think the added encumbrance is worth it.  Your call.